File: AbsManagement_FrontEnd/fe-admin/src/apis/diemDatQuangCao/diemDatQuangCaoModel.ts
interface BaseDiemDatQuangCao {
    diaChi: string;
    phuong: string;
    quan: string;
    danhSachViTri: number[];
    tenLoaiViTri: string;
    tenHinhThucQuangCao: string;
    danhSachHinhAnh: string[];
    idTinhTrang: string;
}

export interface DiemDatQuangCaoModel extends BaseDiemDatQuangCao{
    id: number;
    ngayCapNhat: Date | null;
}

export interface CapNhatDiemDatQuangCaoModel extends BaseDiemDatQuangCao{
    id: number;
    ngayCapNhat: Date | null;
}

export interface ThemDiemDatQuangCaoModel extends BaseDiemDatQuangCao{
}

export interface XoaDiemDatQuangCaoModel{
    id: number;
    ngayCapNhat: Date | null;
}

File: AbsManagement_BackEnd/AbsManagementAPI/Core/Models/DiemDatQuangCao/DiemDatQuangCaoModel.cs
using AbsManagementAPI.Core.Entities;

namespace AbsManagementAPI.Core.Models.DiemDatQuangCao
{
    public class DiemDatQuangCaoModel
    {
        public int Id { get; set; }
        public string DiaChi { get; set; }
        public string Phuong { get; set; }
        public string Quan { get; set; }
        public List<decimal> DanhSachViTri { get; set; }
        public int IdLoaiViTri { get; set; }
        public string TenLoaiViTri { get; set; }
        public int IdHinhThucQuangCao { get; set; }
        public string TenHinhThucQuangCao { get; set; }
        public List<string> DanhSachHinhAnh { get; set; }
        public string IdTinhTrang { get; set; }
        //public virtual ICollection<BangQuangCaoEntity> BangQuangCaos { get; set; }
    }
}
